File: app/src/main/java/com/bakalian/clifford/corners/MainActivity.java
package com.bakalian.clifford.corners;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.StateSet;
import android.view.View;

import view.GameView;


public class MainActivity extends AppCompatActivity {

    public static int num_players = 0;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void startSinglePlayer(View view){
        num_players = 1;
        Intent newGame = new Intent(this, GameActivity.class);
        startActivity(newGame);
    }

    public void startMultiPlayer(View view){
        num_players = 2;
        Intent newGame = new Intent(this, GameActivity.class);
        startActivity(newGame);
    }

    public void settings(View view) {
        Intent settings = new Intent(this, SettingsActivity.class);
        startActivity(settings);
    }

    public void stats(View view){
        Intent stats = new Intent(this, StatActivity.class);
        startActivity(stats);
    }
    public void about(View view) {
        Intent about = new Intent(this, AboutActivity.class);
        startActivity(about);
    }
}

File: app/src/main/java/com/cyb/corners/MainActivity.kt
package com.cyb.corners

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    private fun initButtons(){
        single_button.setOnClickListener {
            val intent = Intent(this, GameActivity::class.java)
            val b = Bundle()
            b.putBoolean("AI",true)
            intent.putExtras(b)
            startActivity(intent)
        }
        multi_button.setOnClickListener {
            val intent = Intent(this, GameActivity::class.java)
            startActivity(intent)
        }
        settings_button.setOnClickListener {
            val intent = Intent(this, SettingsActivity::class.java)
            startActivity(intent)
        }

        help_button.setOnClickListener {
            val intent = Intent(this, AboutActivity::class.java)
            startActivity(intent)
        }
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        supportActionBar?.hide()
        initButtons()
    }

}
